tests/Nox.Generator.Tests/files/yaml/ui/invalid.generator.nox.yaml

```diff
@@ -0,0 +1,25 @@
+﻿#
+# generator.nox.yaml
+#
+# This file configures which layers of code the Nox generator will generate in this project.
+# 
+# Please mark this as a C# analyzer addition file by including the following in your .csproj
+#
+#  <ItemGroup>
+#    <AdditionalFiles Include="generator.nox.yaml" />
+#  </ItemGroup>
+
+# Genrates momain models, events and related types
```

Comment:
Generates domain *
Fix everywhere, please

Reply:
have updated it in all the generator.nox.yaml